Examples/BookingSystem.AspNetCore/Stores/FacilityStore.cs

```diff
@@ -385,6 +385,17 @@ protected override void BookOrderItems(List<OrderItemContext<FacilityOpportunity
                         foreach (var (ctx, id) in ctxGroup.Zip(orderItemIds, (ctx, id) => (ctx, id)))
                         {
                             ctx.SetOrderItemId(flowContext, id);
+
+                            // Setting the access code after booking.
+                            ctx.ResponseOrderItem.AccessCode = new List<PropertyValue>
+                            {
+                                new PropertyValue()
+                                {
+                                    Name = "Pin Code",
+                                    Description = "1234",
```

Comment:
This code should use FakeBookingSystem to return a code that's stored (rather than a hard-coded one), via `BookOrderItemsForFacilitySlot`.

Storing it in FakeBookingSystem will also be useful for SimulateAccessCodeUpdate (Rows 44-45) later.